pywinauto/controls/common_controls.py

```diff
@@ -657,6 +658,44 @@ def deselect(self):
     # Non PEP-8 alias
     Deselect = deselect
 
+    #-----------------------------------------------------------
+    def inplace_control(self, friendly_class_name=""):
+        """Return the editor HwndWrapper of the item
+
+        Possible ``friendly_class_name`` values:
+
+        * ``""``  Return the first appeared in-place control
+        * ``"friendlyclassname"``  Returns editor with particular friendlyclassname
+        """
+        # If currently editing in this item or some other
+        self.listview_ctrl.type_keys("{ENTER}")
+
+        # Get list of visible controls
+        parent_dlg = self.listview_ctrl.top_level_parent()
+        list_before_click = [w.handle for w in parent_dlg.element_info.descendants() if w.visible]
+
+        # After click in the visible list should editable element appears
+        self.click_input(double=True)
+        list_after_click = lambda: [w.handle for w in parent_dlg.element_info.descendants() if w.visible]
+
+        try:
+            chek_func = lambda: len(list_after_click()) > len(list_before_click)

```

Comment:
Antipattern: assigning lambda expression to a variable.
https://docs.quantifiedcode.com/python-anti-patterns/correctness/assigning_a_lambda_to_a_variable.html

Grammar: check_func